File: done/ccc05j2.py
a = int(input())
b = int(input())

print("The number of RSA numbers between", a, "and", b, "is", sum(1 for i in range(a, b + 1) if sum(1 for x in range(1, i+1) if i % x == 0) == 4))

File: done/ccc05j2.cpp
#include <bits/stdc++.h>

using namespace std;
bool RSA(int c) {
    int count;
    count = 0;
    for (int j = 2; j < c; j++) {
        if (c % j == 0) {
            count++;
        }
    }
 if (count == 2) {
     return true;
 }
 return false;
}

int main() {
    int count;
    int a;
    int b;
    cin >> a;
    cin >> b;
    count = 0;
    for (int i = a; i <= b; i++) {
        if (RSA(i)) { 
            count++;
        }
    }
    cout <<"The number of RSA numbers between " << a << " and " << b << " is " << count <<endl;
    return 0;
}